src/osemgrep/reporting/Cli_json_output.mli

```diff
@@ -14,6 +14,7 @@ val exit_code_of_error_type : Semgrep_output_v1_t.error_type -> Exit_code.t
 (* internals used also for incremental display of matches *)
 val cli_match_of_core_match :
   dryrun:bool ->
+  ?applied_fixes:(string, (int * int) list) Hashtbl.t ->
```

Comment:
So this hash is actually populated by the function? it's an OUT parameter?
Would it be possible to return it instead with cli_match?
At least add a comment that this parameter is an OUT parameter modifed by the function
and used in the caller to do ... ?

Reply:
I documented the purpose:
```
   The [applied_fixes] hash table is both in and out parameter, it is used for
   deciding whether a fixed_lines elements is output, depending on whether an
   overlapping fix was already proposed.
```

hope that makes it clear.